src/StylePlugin.js

```diff
@@ -310,7 +286,7 @@ function applyStyleForNextParagraph(prevState, nextState, tr, view) {
           nextNode.attrs.styleName === 'None'
         ) {
           const style = getCustomStyleByName(newattrs.styleName);
-          if (null !== style) {
+          if (null !== style && style.styles) {
```

Comment:
Testing for exactly `null` will cause an exception if style is `undefined`.

`if (style && style.styles)`